### ISSUE TITLE
Do not deploy metrics and tracing Helm charts by default

### ISSUE THREAD

**Issue by kevinpollet** [containous/maesh#684]
### Proposal

Currently, the [metrics](https://github.com/containous/maesh/tree/master/helm/chart/maesh/charts/metrics) and [tracing](https://github.com/containous/maesh/tree/master/helm/chart/maesh/charts/tracing) sub-charts are deployed by default on Maesh installation. The purpose of the Helm chart is to install Maesh and to provide the necessary configuration options to be able to use supported metrics and tracing library, and not to provide production-ready stacks for metrics and tracing.

Therefore, the metrics and tracing sub-charts must not be deployed by default but should be kept for testing and demonstration purposes.

**Comment by kevinpollet** [comment]
@SantoDE @jspdown Thoughts?

**Comment by jspdown** [comment]
I agree with this proposal but it's remains important to provide working and meaningful configurations for metrics and tracing. I suggest to put these files in a `contrib` folder like on [Traefik](https://github.com/containous/traefik/tree/master/contrib)

**Comment by kevinpollet** [comment]
For now, I would just keep the Helm chart as is and disable the deployment by default. I also agree that we should find a way to distribute configurations for common tools, like `grafana`.
